File: KuantumKaosOdev-java/src/AntiMadde.java
public class AntiMadde extends KuantumNesnesi implements IKritik {

    public AntiMadde(String id) {
        super(id, 10);
    }

    @Override
    public void AnalizEt() throws KuantumCokusuException {
        double yeniDeger = getStabilite() - 25;
        setStabilite(yeniDeger);
        System.out.println("Evrenin dokusu titriyor...");
    }

    @Override
    public void AcilDurumSogutmasi() {
        try {
            double yeniDeger = getStabilite() + 50;
            setStabilite(yeniDeger);
            System.out.println(getId() + " üzerine acil durum soğutucusu sıkıldı!");
        } catch (KuantumCokusuException e) {
        }
    }
}

File: KuantumKaosOdev-csharp/AntiMadde.cs
using System;

namespace KuantumKaos
{
    public class AntiMadde : KuantumNesnesi, IKritik
    {
        public AntiMadde(string id) : base(id, 10) // Maksimum tehlike
        {
        }

        public override void AnalizEt()
        {
            Stabilite -= 25;
            Console.WriteLine("Evrenin dokusu titriyor...");
        }

        public void AcilDurumSogutmasi()
        {
            Stabilite += 50;
            Console.WriteLine($"{ID} üzerine acil durum soğutucusu sıkıldı!");
        }
    }
}